.ci/openshift-ci/build-root/scripts/qs-overrides/opentelemetry-tracing/overridable-functions.sh

```diff
@@ -0,0 +1,9 @@
+function runPostHelmInstallCommands() {
+  echo "Applying all OpenTelemetry Collector resources"
+  oc apply -f charts/opentelemetry-collector.yaml
+}
+
+function cleanPrerequisites() {
+  echo "Deleting all OpenTelemetry Collector resources"
+  oc delete -f charts/opentelemetry-collector.yaml
```

Comment:
I think this step should as additional step in undeploy

Reply:
The proper solution for this requires an additional extension point for shared-docs, similar to helm-install-prerequisites, and that will affect all QS so I have created https://issues.redhat.com/browse/WFLY-18986 to deliver it (and more general shared improvements). For now the uninstall is referenced as a note at https://github.com/wildfly/quickstart/pull/872/files#diff-437f0901bdbc94e992d4b6b75f355240a557c4879683f47d27dad1c51f27dc4aR19